metadata/metadata.go

```diff
@@ -191,6 +258,42 @@ func (m *client) GetStacks() ([]Stack, error) {
 	return stacks, nil
 }
 
+func (m *client) GetEnvironments() ([]Environment, error) {
+	var environments []Environment
+	resp, err := m.SendRequest("/environments")
+	if err != nil {
+		return environments, err
+	}
+	if err = json.Unmarshal(resp, &environments); err != nil {
+		return environments, err
+	}
+	return environments, nil
+}
+
+func (m *client) GetServicesFromRegionEnvironment(regionName string, envName string) ([]Service, error) {
+	environments, err := m.GetEnvironments()
+	var services []Service
+	if err != nil {
+		return services, err
+	}
+
+	for _, env := range environments {
+		if regionName == env.RegionName && envName == env.Name {
+			services = append(services, env.Services...)
```

Comment:
@kinarashah you should return right here, no need to iterate over the rest of the environments once found